tools/python/util/convert_onnx_models_to_ort.py

```diff
@@ -239,13 +235,23 @@ def convert_onnx_models_to_ort():
     else:
         session_options_config_entries["session.qdqisint8allowed"] = "0"
 
-    for optimization_level in args.optimization_level:
-        print(f"Converting models and creating configuration file for optimization level '{optimization_level}'")
-        _convert(model_path_or_dir, optimization_level, args.use_nnapi, args.use_coreml, custom_op_library,
+    for optimization_style in optimization_styles:
+        print("Converting models for optimization level '{}' and style '{}'".format(
+            optimization_level_str, optimization_style.name))
+
+        _convert(model_path_or_dir, optimization_level_str, optimization_style, custom_op_library,
                  args.save_optimized_onnx_model, args.allow_conversion_failures, args.target_platform,
                  session_options_config_entries)
 
-        _create_config_file_from_ort_models(model_path_or_dir, optimization_level, args.enable_type_reduction)
+        if optimization_style == OptimizationStyle.Runtime:
+            print("Config file generation is not supported for ORT format models with runtime optimizations. "
```

Comment:
>Config file generation is not supported for ORT format models with runtime optimizations [](http://example.com/codeflow?start=19&length=88)

realized that the config file generation doesn't handle runtime optimizations, so just disabling it

Reply:
handling this case by generating a config file from models with and without runtime optimizations